src/settings.plugin.js

```diff
@@ -203,18 +238,19 @@ wikiaJWPlayerSettingsPlugin.prototype.updateQualityLevelsList = function (qualit
 		}
 
 		qualityLevelItem.appendChild(document.createTextNode(level.label));
-		qualityLevelsList.insertBefore(qualityLevelItem, backButton);
+		this.qualityLevelsList.insertBefore(qualityLevelItem, this.backButton);
 	}, this);
 };
 
-wikiaJWPlayerSettingsPlugin.prototype.updateCurrentQuality = function (qualityLevelsList, isActiveClass, data) {
-	qualityLevelsList.childNodes.forEach(function (node, index) {
-		if (data.currentQuality === index) {
+wikiaJWPlayerSettingsPlugin.prototype.updateCurrentQuality = function (data) {
+	for (var i = 0; i < this.qualityLevelsList.childNodes.length; i++) {
```

Comment:
why did you changed that?

\+ this makes our polyfill redundant

Reply:
I'll remove the polyfill in App. We want to support IE11 without any dependencies needed.